Tests/TestHelpers/TestResources.swift

```diff
@@ -2,14 +2,14 @@ import Foundation
 import SwiftLintCore
 
 public enum TestResources {
-    public static func path(_ calleePath: String = #filePath) -> String {
+    public static func path(_ calleePath: String = #filePath) -> URL {
```

Comment:
Can symlinks be resolved once here? I assume symlinks appear when running with Bazel only?